geomesa-utils/src/main/scala/org/locationtech/geomesa/utils/geotools/Conversions.scala

```diff
@@ -191,6 +193,11 @@ object RichSimpleFeatureType {
   val DEFAULT_DATE_KEY    = "geomesa.index.dtg"
   val ST_INDEX_SCHEMA_KEY = "geomesa.index.st.schema"
   val USER_DATA_PREFIX    = "geomesa.user-data.prefix"
+  val KEYWORDS_KEY        = "geomesa.keywords"
+
+  val KEYWORDS_SPLITTER = "\\|"
```

Comment:
I would just keep it as a comma everywhere unless we have a compelling use case not to... otherwise we are going to have to do a lot of special handling to check for commas and replace them, etc - plus if we're using commas in the input strings, doesn't that imply that restrict us from having commas in the list values?